slither/__main__.py

```diff
@@ -402,6 +402,14 @@ def parse_args(detector_classes, printer_classes):  # pylint: disable=too-many-s
         default=defaults_flag_in_config["fail_pedantic"],
     )
 
+    group_detector.add_argument(
+        "--no-fail-pedantic",
+        help="Don't fail immediately if a finding is detected",
```

Comment:
Maybe `Do not return the number of findings in the exit code?`, or something similar?

We could update the helper for `-fail-pendantic` to make it clear that both are related.

Slither does not fail immediately if this flag is not enabled ;)

Reply:
Ok. I based this on the previous help sentence for `--fail-pedantic`, we can change it